### DISCUSSION TITLE
Is it possible to support MySQL behind a proxysql?

### DISCUSSION THREAD

**Discussion by ErikXu** [apache/superset#19591] [Q&A / Help]
I've setup a MySQL cluster behind a [proxysql](https://proxysql.com/) - A high performance MySQL proxy.
I can't use the proxysql connection string to connect MySQL in superset, while using MySQL server IP works. 
Is it possible to connect superset to MySQL behind a proxysql? 
Thanks!

**Comment by villebro** [comment]
The `proxysql` homepage states that "ProxySQL understands the MySQL protocol and features advanced connection handling capabilities", which to me seems like it should be possible to just replace the MySQL server IP with the ProxySQL one. Is this not the case?

**Comment by ErikXu** [comment]
With no luck, it's not the case. The database cannot be added via ProxySQL IP.
